packages/flutter_reactive_ble/pubspec.lock

```diff
@@ -371,16 +371,16 @@ packages:
   reactive_ble_mobile:
     dependency: "direct main"
     description:
-      name: reactive_ble_mobile
-      url: "https://pub.dartlang.org"
-    source: hosted
+      path: "../reactive_ble_mobile"
```

Comment:
can you revert this one and the package below?

Reply:
All pubspec.lock files are now reverted :-)